CHANGELOG.md

```diff
@@ -1,5 +1,111 @@
 # Changelog
 
+## v6.0.0-beta.8 - 2017-10-05
+
+### Engine updates and fixes
```

Comment:
We need to have the `Breaking Changes` on top: #4802, #4866, #4903, #4963

Reply:
resolved